README.md

```diff
@@ -386,12 +386,25 @@ Or with JUnit 5 :
 
 ```
 @ExtendWith(LocalstackDockerExtension.class)
-@LocalstackDockerProperties(randomizePorts = true, services = { "sqs", "kinesis:77077" })
+@LocalstackDockerProperties(services = { "sqs", "kinesis:77077" })
 public class MyDockerCloudAppTest {
    ...
 }
 ```
 
+You can configure the Docker behaviour using the `@LocalstackDockerProperties` annotation with the following parameters:
+
+| property                    | usage                                                                                                                        | type                         | default value |   |
```

Comment:
nit: seems like there is an unused additional column on the right side of the table which can be removed